plugins/multiforward.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: multiforward
+spec:
+  version: v0.0.0
+  homepage: https://github.com/njnygaard/kubectl-multiforward
+  shortDescription: Port Forward to multiple Kubernetes Services
+  description: |
+    This plugin allows you to forward to multiple Kubernetes Services
+    without the use of multiple terminal windows.
+  caveats: |
+    * You should have ~/.multiforward.yaml or ./.multiforward.yaml configured
```

Comment:
Please add instructions how to find out about the format for this config file. For example, give a link (please point to HEAD and not to a specific version), or maybe `multiforward --help` will do?

Reply:
I've added to the `Long` help, let me know if you think it is _too_ verbose now. 

```plaintext
❯ kubectl multiforward --help
An example format can be found here: https://github.com/njnygaard/kubectl-multiforward/blob/master/.multiforward.example.yaml

Or you can reference the following YAML structure where 'displayName' is a human-readable handle and 'name' and 'namespace' identify the Kubernetes resource.

groups:
  - name: production
    services:
      - displayName: Alertmanager
        port: 9093
        namespace: monitoring-prometheus
        name: alertmanager-operated
        protocol: http
      - displayName: Prometheus
        port: 9090
        namespace: monitoring-prometheus
        name: prometheus-operated
        protocol: http
  - name: staging
    services:
      - displayName: Elasticsearch
        port: 9200
        namespace: monitoring-eck
        name: elasticsearch-es-http
        protocol: https

Usage:
  multiforward <group> [flags]

Examples:
kubectl multiforward [group]

Flags:
  -h, --help   help for multiforward
```

Reply:
Yes looks good to me. But can you also mention here that `--help` has instructions for the config file?
Imagine that your users have 20s to find out if this plugin is a fit and how to use it. The more effort it takes to get it running the fewer users you will have.

Reply:
@njnygaard I think you might have missed this comment, because I forgot to unresolve.

Reply:
> But can you also mention here that --help has instructions for the config file?

Where is the 'here' we're talking about? in the output of `Flags:` @corneliusweig ?

Reply:
@corneliusweig I believe I understand, I've added this as the default zero argument command for the `v0.0.3` release I just pushed. I've updated this accordingly.

```plaintext
$ ./kubectl-multiforward

An example format for your local config file (~/.multiforward.yaml or .multiforward.yaml) can be found here: https://github.com/njnygaard/kubectl-multiforward/blob/master/.multiforward.example.yaml

Or you can reference the following YAML structure where 'displayName' is a human-readable handle and 'name' and 'namespace' identify the Kubernetes resource.

groups:
  - name: production
    services:
      - displayName: Alertmanager
        port: 9093
        namespace: monitoring-prometheus
        name: alertmanager-operated
        protocol: http
      - displayName: Prometheus
        port: 9090
        namespace: monitoring-prometheus
        name: prometheus-operated
        protocol: http
  - name: staging
    services:
      - displayName: Elasticsearch
        port: 9200
        namespace: monitoring-eck
        name: elasticsearch-es-http
        protocol: https

Usage:
  multiforward <group> [flags]

Examples:
kubectl multiforward [group]

Flags:
  -h, --help   help for multiforward, contains config locations and sample configuration

Error: please specify a group found in your config (for you, one of these choices: "production, staging")
```